lib/web/websocket/frame.js

```diff
@@ -27,7 +27,7 @@ try {
 function generateMask () {
   if (bufIdx === BUFFER_SIZE) {
     bufIdx = 0
-    crypto.randomFillSync((buffer ??= Buffer.allocUnsafe(BUFFER_SIZE)), 0, BUFFER_SIZE)
+    crypto.randomFillSync((buffer ??= Buffer.allocUnsafeSlow(BUFFER_SIZE)), 0, BUFFER_SIZE)
```

Comment:
No problem, but with the current code, I have some concerns.

Reply:
I don't see them, unless we acess buffer.buffer anywhere

Reply:
If they access it, they can rewrite the mask, right?

Reply:
example:
```js
import { WebSocket } from './index.js'

Buffer.poolSize = 16386 * 4

// any server url
const u = new WebSocket('ws://localhost:3000')

u.onmessage = (data) => {
  // The mask can be rewritten here.
  new Uint8Array(Buffer.allocUnsafe(1).buffer).fill(0)
  u.send('Hi')
}
u.onopen = () => {
  u.send('Hi')
}

```